src/sentry/remote_config/docs/api.md

```diff
@@ -53,105 +39,231 @@ Retrieve the project's configuration.
       "options": {
         "sample_rate": 1.0,
         "traces_sample_rate": 0.5
+      },
+      "version": 1
+    }
+    ```
+
+## Configurations [/projects/<organization_id_or_slug>/<project_id_or_slug>/configurations/]
+
+| Field              | Type             | Description                                                                   |
+| ------------------ | ---------------- | ----------------------------------------------------------------------------- |
+| environment        | optional[string] | The environment the configuration is associated with.                         |
+| id                 | string           | A server generated unique identifier.                                         |
+| name               | optional[string] | A custom name distinguishing the configuration from the default project name. |
+| sample_rate        | number           |                                                                               |
+| traces_sample_rate | number           |                                                                               |
+| version            | number           | The current configuration version. Initialized to 1.                          |
+
+### Get Configurations [GET]
+
+Retrieve configurations.
+
+- Response 200
+
+  ```json
+  {
+    "data": [
+      {
+        "environment": null,
+        "id": "0b88ac27a7b444a6baeb312c0493aed5",
+        "name": null,
+        "sample_rate": 0,
+        "traces_sample_rate": 0.75,
+        "version": 1
       }
+    ]
+  }
+  ```
+
+### Create Configuration [POST]
+
+Create a new configuration. The version field is ignored on create. Version is initialized to 1.
+
+- Request
+
+  ```json
+  {
+    "data": {
+      "environment": "production",
+      "name": "custom-name",
+      "sample_rate": 1.0,
+      "traces_sample_rate": 0.5
+    }
+  }
+  ```
+
+- Response 201
+
+  ```json
+  {
+    "data": {
```

Comment:
the GET is returning `data: [ {...}, {...} ]` (arraY) and the POST returns `data: { ... }` (same object as `GET /projects/<organization_id_or_slug>/<project_id_or_slug>/configurations/<configuration_id>/` it seems)

Just writing for clarity.